### ISSUE TITLE
IE8 error

### ISSUE THREAD

**Issue by armandn** [braintree/braintree-web#141]
IE 8 produces this error upon setup:

SCRIPT438: Object doesn't support this property or method
braintree-2.21.0.min.js (5,26658)

Works in IE9+ and other browsers.

Here's my init object:
```
braintree.setup(
CLIENT_TOKEN,
"custom",
{
	id: "paymentForm",
	onReady: onFormReady,
	onPaymentMethodReceived: onNonce,
	onError: onError,
	hostedFields:
	{
		number:
		{
			selector: "#number"
		},
		cvv:
		{
			selector: "#cvv"
		},
		expirationMonth:
		{
			selector: "#expirationMonth",
			placeholder: "MM"
		},
		expirationYear:
		{
			selector: "#expirationYear",
			placeholder: "YY"
		},
		styles:
		{
			"input":
			{
				"font-family": "Tahoma, Arial, sans-serif",
				"font-size"  : "16px",
				"color"      : "#000"
			}
		},
		onFieldEvent: onFieldEvent
	},
	paypal:
	{
		singleUse     : true,
		amount        : PRODUCT_PRICE,
		currency      : PRODUCT_CURRENCY,
		headless      : true,
		displayName   :PRODUCT_NAME,
		onUnsupported : onPayPalUnsupported,
		onSuccess     : onPayPalSuccess
	},
	dataCollector:
	{
		kount: {environment: ENVIRONMENT}
	}
})
```

**Comment by EvanHahn** [comment]
We looked into this and found that an error is thrown in IE8 but everything should be working fine. It's still a problem on our end, but your checkout should function properly.

We'll update you when we have a new version that removes this error.

**Comment by EvanHahn** [comment]
This has been fixed in `braintree-web@2.22.2`. You can grab it from Bower or npm, or just link to the file at `https://js.braintreegateway.com/js/braintree-2.22.2.min.js`.

Thanks for reporting!
